espnet2/tasks/hubert.py

```diff
@@ -282,8 +283,37 @@ def build_collate_fn(
             pad=args.collate_fn_conf.get("pad", False),
             rand_crop=args.collate_fn_conf.get("rand_crop", True),
             crop_audio=not args.collect_stats,
+            kernel_size=args.collate_fn_conf["kernel_size"],
+            stride=args.collate_fn_conf["stride"],
+            sample_rate=args.collate_fn_conf["sample_rate"],
         )
 
+    @classmethod
+    def update_collate_info(cls, args: argparse.Namespace) -> None:
+        assert check_argument_types()
+
+        fs = args.frontend_conf.get("fs", None)
```

Comment:
Ditto. Would there be a risk of using previous configs? We used to set
```
frontend: null
```

Reply:
This is OK. SInce the frontend_conf would be still loaded as `{}` even if the frontend is null.